enterprise-suite/values.yaml

```diff
@@ -27,8 +27,8 @@ busyboxVersion: "1.30"
 #
 # es-console image
 esConsoleImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/es-console'
-# es-monitor-api image
-esMonitorImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/es-monitor-api'
+# console-api image
+esMonitorImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/console-api'
```

Comment:
Do we want to deprecate this and use `consoleImage` here?  (And ditto for `esMonitorVersion` -> `consoleVersion` above?)

Reply:
I think that should be a separate issue - to remove all the `es` prefixes.